qds_sdk/qubole.py

```diff
@@ -21,6 +21,8 @@ class Qubole:
     """
 
     MIN_POLL_INTERVAL = 1
+    RETRIES_CAP = 6
```

Comment:
Calculated the values as follow- 
With backoff factor as 2 and retry count of 6, we will be retrying 5 times, till the retry_count is >1, decrementing it on each retry. Keeping base_delay as 10sec,
Thus, all retries will be completed within
10 + 20 + 40 + 80 + 160 = 310sec (around 5 minutes)